setup.py

```diff
@@ -40,6 +40,18 @@ def _find_msbuild_tool(tool="msbuild.exe", use_windows_sdk=False):
                 locappdir, vcpy27, r"x64\mt.exe")
             if os.path.exists(mtpath):
                 return mtpath      
+        if sys.version_info[:2] == (3,5):
```

Comment:
removing from rebase. agree with @tonyroberts 
@denfromufa may need to add it to the registry.